### ISSUE TITLE
[Feature] Other Kinesis shard iterator types

### ISSUE THREAD

**Issue by m-radzikowski** [dinsaw/kines#8]
Hey,
it's a really useful tool!

What I am missing is the ability to debug stream in real time, needed during the development. There is a `walk` command, but it supports only two iterator types - `TRIM_HORIZON` (default) and `AT_SEQUENCE_NUMBER`, right?

Would be great to support also `AT_TIMESTAMP`, maybe be providing full datetime or time description like `10 minutes ago`?

Also would be nice to have long-living polling - start the tool with `LATEST` iterator and just get all new incoming events (by calling Kinesis every second or two).

What do you think?

**Comment by dinsaw** [comment]
Introducing new options in `walk` command.
- [x] `-f` option to fetch records continuously within some time gap.
- [x] `-t` option for `AT_TIMESTAMP` iterator
- [x] `-l` option for `LATEST` iterator

**Comment by dinsaw** [comment]
Released new version `1.1.0` with following changes.
```
Usage: kines walk [OPTIONS] STREAM_NAME SHARD_ID

  Walk through Kinesis Records. By default creates TRIM_HORIZON iterator.

Options:
  -t, --timestamp TEXT            Start from this timestamp. Creates
                                  AT_TIMESTAMP shard iterator. You can specify
                                  timestamp as '2016-04-04T19:58:46.480-00:00'
                                  or with short hand as '1h10m' i.e. 1 hour 10
                                  minutes ago, '10m' i.e. 10 minutes ago, '1h'
                                  i.e. 1 hour ago.

  -l, --latest                    Start from latest records. Creates LATEST
                                  shard iterator

  -f, --follow                    Poll records repeatedly after 2 seconds
  -s, --sequence-number TEXT      Start from this Sequence Number. Creates
                                  AT_SEQUENCE_NUMBER shard iterator.

  -n, --number-of-records INTEGER
                                  Max Number of records to be retrieved per
                                  call

  --help                          Show this message and exit.

```
